distributed/cli/dask_scheduler.py

```diff
@@ -111,6 +112,15 @@ def del_pid_file():
         limit = max(soft, hard // 2)
         resource.setrlimit(resource.RLIMIT_NOFILE, (limit, hard))
 
+    # check if DASK_SCHEDULER_IP and DASK_SCHEDULER_PORT are defined
+    if not host:
+        dict_view = config.keys
+        if PY2:
+            dict_view = config.viewkeys
+        if {'scheduler-ip', 'scheduler-port'} <= dict_view():
```

Comment:
The above seems more complex than it needs to be.  If I were to write this I would say `if 'scheduler-ip' in config and 'scheduler-port' in config:`